agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/statsbeat/NetworkStatsbeat.java

```diff
@@ -42,6 +44,9 @@
   private static final String EXCEPTION_COUNT_METRIC_NAME = "Exception Count";
   private static final String BREEZE_ENDPOINT = "breeze";
 
+  private static final Pattern hostPattern =
+      Pattern.compile("^(?:https?://)?(?:www\\.)?([^/\\.]+)");
```

Comment:
do we need to support url with missing protocol (no `http://` or `https://`)? does IngestionEndpoint support that?

Reply:
this pattern works for all cases.